lib/features/give/bloc/give/give_bloc.dart

```diff
@@ -554,6 +562,10 @@ class GiveBloc extends Bloc<GiveEvent, GiveState> {
     if (mediumId.split('.').last.contains('b6')) {
       return;
     }
+    if (mediumId.split('.').last.startsWith('6')) {
```

Comment:
Why?

Reply:
Check slack. I message you the reason. Also, i feel like it's an implementation bottleneck that we would have to deal with in the future as they have a designated instance type in the medium id